plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -420,6 +422,13 @@ private function get_paypal_create_order_request_params( $order, $payment_source
 			}
 		}
 
+		$order_items = $this->get_paypal_order_items( $order );
+		if ( empty( $order_items ) ) {
+			// If we cannot build order items (e.g., negative item amounts),
```

Comment:
Nit: the `,` is not needed after the `e.g.`
```suggestion
			// If we cannot build order items (e.g. negative item amounts),
```